pl_bolts/datamodules/vocdetection_datamodule.py

```diff
@@ -0,0 +1,199 @@
+import torch
```

Comment:
no need to repeat `_datamodule.py` in the name

Reply:
I figured I would name it to be consistent with `imagenet_datamodule.py`, `mnist_datamodule.py`, etc. We may just want to rename all of these in the future.

Reply:
i agree with consistency and yes, we need to keep the _datamodule.py name

Reply:
well, let's talk about refactoring later...